lib/cli.ts

```diff
@@ -169,22 +203,37 @@ export class Cli {
           'Log level (error, warn, info, verbose, debug)',
           this.args.logLevel,
         )
+        .option(
+          'noAlias',
+          'Don\'t add an alias for the main NPM package file to the generated .d.ts source',
+          this.args.noAlias,
+        )
         .option(
           ['f', 'force'],
           'Ignores non-critical errors and attempts to at least partially generate typings',
           this.args.force,
         )
+        .option(
+          'shake',
+          'Basic tree-shaking for modules. (off (default), exportOnly, allImports). Drops modules not referenced by entry. exportOnly only keeps modules which are referenced with the export from ... keyowrd.',
+          this.args.shake,
+        )
         .option(
           ['o', 'output'],
           'Overrides recommended output target to a custom one',
           this.args.output,
         )
+        .option(
+          'template',
+          'Append this template where {0} is replaced with the name/path of the entry module.',
+          this.args.template,
+        )
         .option(
           ['m', 'testMode'],
           'Configures npm-dts for self-test',
           this.args.testMode,
         )
-        .command('generate', 'Start generation', (name, sub, options) => {
+        .command('generate', 'Start generation', (name: any, sub: any, options: any) => {
```

Comment:
any should not be required. Types are: `string`, `string[]` and `ConfigurationOptions`. Did you have some issue with those types?